frontend/wallet/src/render/DownloadLogic.re

```diff
@@ -1,22 +1,27 @@
 module Filestream = {
```

Comment:
Should this file be in render?
It also looks like already have some stream bindings (though pretty bare-bones) in https://github.com/CodaProtocol/coda/blob/643e7a1ee85b8a79aa8cda4f4658b0d6668728b8/frontend/wallet/src/common/Bindings.re#L57-L66
Can we update those instead of having them in two places? The bindings you have look fine, but maybe we can update the other spots to use them, if it's not too much trouble

Reply:
Totally yeah.